### PR TITLE
Continued attempts to get CI passing

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

**Comment by mikalstill** [comment]
Ignoring https://github.com/shakenfist/shakenfist/issues/359 occassionally popping up and GCP sometimes giving us nodes so slow things timeout, this appears to be reliable now. Reviews appreciated.

File: shakenfist/config.py [hunk 1]
```diff
@@ -137,6 +137,8 @@ def parse(self):
                     value = int(value)
                 elif isinstance(CONFIG_DEFAULTS[flag], float):
                     value = float(value)
+                elif isinstance(CONFIG_DEFAULTS[flag], bool):
+                    value = (value == 'true')
```
Comment (mandoonandy) [inline shakenfist/config.py:141]: Given that some people may use `True`, maybe
```
from distutils.util import strtobool
value = strtobool(value)
```

**Review by mandoonandy — CHANGES_REQUESTED** [review]
One annoying suggestion because I know you love them.

File: shakenfist/config.py (hunk 1 above)
Comment (mikalstill) [reply shakenfist/config.py:141]: Done.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**